scripts/masterCron.ts

```diff
@@ -354,8 +354,9 @@ async function settleAndBurn(
           .settleAuction(auctionID.toString(), {
             gasPrice: newGasPrice,
             gasLimit: gasLimits["settleAuction"],
-          })
-          .wait();
+          });
+
+        await tx.wait();
```

Comment:
Separating out wait ensures it throws with the correct message if settleAuction fails